index.html

```diff
@@ -0,0 +1,342 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta
+        name="viewport"
+        content="width=device-width, initial-scale=1.0"
+    >
+    <link
+        rel="preconnect"
+        href="https://fonts.googleapis.com"
+    >
+    <link
+        rel="preconnect"
+        href="https://fonts.gstatic.com"
+        crossorigin
+    >
+    <link
+        href="https://fonts.googleapis.com/css2?family=Montserrat:wght@100..900&display=swap"
+        rel="stylesheet"
+    >
+    <link
+        rel="stylesheet"
+        href="./css/style.css"
+    >
+    <title>Nova template</title>
+</head>
+
+<body>
+
+    <header class="header">
+
+        <section class="header__container container">
+
+            <figure class="header__logo">
+                <img
+                    class="logo__image"
+                    src="./images/logo.svg"
+                    alt="Logo image"
+                >
+            </figure>
+
+            <nav class="header__nav">
+                <ul class="nav__links">
+                    <a href="./#team">
```

Comment:
Raczej zrobiłbym na odwrót czyli `<li><a>`, bo element typu `inline` nie powinien zawierać elementu blokowego, ale rozumiem, z czego to wynika ;)